test/CheckoutTest.py

```diff
@@ -97,7 +97,7 @@ def test_payments_error_mocked(self):
                 'returnUrl': 'https://your-company.com/...',
                 u'applicationInfo': {
                     u'adyenLibrary': {
-                        u'version': '6.0.0',
+                        u'version': '7.0.0',
```

Comment:
Can you please merge `develop` into your branch and bump this version(s)?
```suggestion
                        u'version': '7.1.0',
```